src/autocomplete.js

```diff
@@ -18,6 +18,13 @@ const hasPointerEvents = (() => {
   return element.style.pointerEvents === 'auto'
 })()
 
+const createSimpleEngine = results => (query, syncResults) => {
```

Comment:
Later (not now) we will want to put this into its own module and write some tests for it.